### PR TITLE
Dump mapping table

### PR DESCRIPTION
**Pull request by deep-diver**

### CODE REVIEW

**Comment by deep-diver** [comment]
디렉터리 구조를 어떻게 잡아야 할지 몰라서 `cmd/tools` 아래에 넣었습니다.

File: server/pkg/pr12er/database.proto [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+message MappingTable {
+  repeated MappingTableRow rows = 1;
+}
+
+message MappingTableRow {
+  int32 pr_id = 1;
+  repeated string paper_arxiv_id = 2;
+  string youtube_video_id = 3;
+//  repeated Override overrides = 4;
```
Comment (kkweon) [inline server/pkg/pr12er/database.proto:17]:
```suggestion
```

File: server/cmd/tools/mappingtable/mapping_table_creation.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	marshalOption := prototext.MarshalOptions{Multiline: true}
+	bytes, err := marshalOption.Marshal(&mappingTable)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	_, err = f.Write(bytes)
+	if err != nil {
+		log.Fatal(err)
+	}
```
Comment (kkweon) [inline server/cmd/tools/mappingtable/mapping_table_creation.go:63]: File was not closed
```suggestion
	f, err := os.Create("test.pbtxt")
	if err != nil {
		log.Fatal(err)
	}
	defer f.Close()

	marshalOption := prototext.MarshalOptions{Multiline: true}
	bytes, err := marshalOption.Marshal(&mappingTable)
	if err != nil {
		log.Fatal(err)
	}

	_, err = f.Write(bytes)
	if err != nil {
		log.Fatal(err)
	}
```

or simply

```suggestion
	marshalOption := prototext.MarshalOptions{Multiline: true}
	bytes, err := marshalOption.Marshal(&mappingTable)
	if err != nil {
		log.Panic(err)
	}
	if err = ioutil.WriteFile("test.pbtxt", bytes, 0644); err != nil {
		log.Panic(err)
	}
```

File: server/pkg/env/env.go [hunk 3]
```diff
@@ -10,6 +10,8 @@ var (
 	ServicePort = getEnvVar("PR12ER_GRPC_PORT", "9000")
 	// PrometheusPort is the port for Prometheus metrics (/metrics).
 	PrometheusPort = getEnvVar("PR12ER_PROMETHEUS_PORT", "9092")
+	// GoogleSearchAPIKey is the api key for calling searchArxivID function.
+	GoogleSearchAPIKey = getEnvVar("GOOGLE_SEARCH_API_KEY", "NONE")
```
Comment (kkweon) [inline server/pkg/env/env.go:14]: Do not define non Server env.

Instead, define in the main package.

```suggestion
```

File: server/pkg/pr12er/database.proto (hunk 1 above)
Comment (deep-diver) [reply server/pkg/pr12er/database.proto:17]: Override is not defined yet, so I have commented it out

File: server/cmd/tools/mappingtable/mapping_table_creation.go (hunk 2 above)
Comment (deep-diver) [reply server/cmd/tools/mappingtable/mapping_table_creation.go:63]: Close를 해야했군요 이런 기본적인 실수를...

**Comment by nicewook** [comment]
LGTM! 
I will make CLI from this PR as a base.

**Pull request merged**